src/app/coursegrab/dao/courses_dao.py

```diff
@@ -19,3 +19,8 @@ def create_course(subject_code, course_num, title):
 
 def search_courses(query):
     return Course.query.filter(Course.search_string.ilike("%{}%".format(query))).limit(20)
+
+
+def clear_table():
+    Course.__table__.drop(db.engine)
+    Course.__table__.create(db.engine)
```

Comment:
Rather than dropping and recreating the table I think you can just do`Course.query.delete()` to delete all rows